src-tauri/src/tray/tray.rs

```diff
@@ -81,6 +82,11 @@ fn add_node_items(menu: SystemTrayMenu, node: &str, is_running: bool) -> SystemT
     } else {
         CustomMenuItem::new(format!("stop_{}", node), "Stop").disabled()
     })
+    .add_item(if is_running {
+        CustomMenuItem::new(format!("dashboard_{}", node), "Dashboard")
+    } else {
+        CustomMenuItem::new(format!("dashboard_{}", node), "Dashboard").disabled()
+    })
```

Comment:
works, but you can avoid duplication

```suggestion
    .add_item({
        let item = CustomMenuItem::new(format!("dashboard_{}", node), "Dashboard");
        is_running.then(|| item).unwrap_or_else(|| item.disabled())
    })
```